setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
```

Comment:
This is not a robust pattern for packaging.  Sometimes the distribution will be compressed into an archive format like .egg or .whl for Python, and there is no file on disk to be resolved.  The preferred method is to use the *pkg_resources* module from setuptools (Python 3.6) or *importlib.resources* (Python 3.7+).